### ISSUE TITLE
How to collaborate if all the posts are answered...

### ISSUE THREAD

**Issue by susanBuck** [susanBuck/dwa15-fall2017#37]
Hi everyone,

In the progress logs some of you have noted in the *Collaboration* section that you weren't able to contribute because...

+ You didn't have a question
+ The questions that were posted were already closed (i.e. resolved)
+ There were unresolved questions but you weren't sure how to help

If this is the case, take the time to skim through some of the closed posts from the week. 

Is there anything in these posts that...

+ Taught you something you didn't already know?
+ Mentioned something you didn't understand?
+ Describe a problem you also experienced?
+ Etc.

Another way you can contribute is giving a "thumbs up" to existing student answers if you feel they best address the question at hand. This helps other students who are skimming those threads to quickly pick out the optimal answer.

__In short, just give me some indication that you're engaging with the content here in the Issues forum and you'll receive the full points for Collaboration.__

**Comment by sheetaldalvi** [comment]
Thanks so much for this clarification.
